wildstore-cli/src/main/java/edu/sjsu/wildstore/Main.java

```diff
@@ -123,32 +123,31 @@ public void search(@CommandLine.Parameters(paramLabel = "query") String query,
         }
 
         @CommandLine.Command
-        public void clean(@CommandLine.Parameters(paramLabel = "limit") int limit,
-                          @CommandLine.Parameters(paramLabel = "hostname") String hostname,
-                          @CommandLine.Option(names = "--token") String token,
-                          @CommandLine.Option(names = "--dryrun", negatable = true, defaultValue = "true")
-                          boolean dryrun) throws InterruptedException, ExecutionException {
+        public void clean(@CommandLine.Mixin CliOptions co,
+                          @CommandLine.Parameters(paramLabel = "limit", defaultValue = "10000") int limit,
+                          @CommandLine.Option(names = "--dryrun", negatable = true) Boolean dryrun)
+                throws InterruptedException, ExecutionException {
             int offset = 0;
             LinkedMultiValueMap<String, String> parameters = new LinkedMultiValueMap<String, String>();
             parameters.add("limit", String.valueOf(limit));
             List<String> result;
             int i = 0;
-            WebClient webClient = Client.getWebClient(hostname + "/api/metadata/filepath");
+            WebClient webClient = Client.getWebClient(co.metadataURL + "/api/metadata/filepath", co.token);
+            boolean dryrunVal = (dryrun != null) ? dryrun : true;
 
             do {
                 parameters.put("offset", List.of(String.valueOf(offset)));
                 result = (List<String>) Client.get(webClient,
                                                    parameters,
                                                    new ParameterizedTypeReference<List<String>>() {});
-                System.out.println("The following Metadata documents will be removed from the database:");
-                result.forEach(System.out::println);
                 List<String> deletedFiles = result.stream().filter(item -> !Files.exists(Paths.get(item))).toList();
-                System.out.println("DELETE RESULT:" + Client.post(webClient,
+                if (!dryrunVal) System.out.println("DELETE RESULT:" + Client.post(webClient,
                                                                   deletedFiles,
                                                                   new ParameterizedTypeReference<Integer>() {},
-                                                                  httpHeaders -> httpHeaders.setBearerAuth(token)));
+                                                                  httpHeaders -> httpHeaders.setBearerAuth(co.token)));
+                co.out().println("Deleted Files: " + String.join("\n", deletedFiles));
                 offset += limit;
-            } while (!result.isEmpty());
+            } while (result.size() == offset);
```

Comment:
at the end if dryrun == true you should print the message like "DRYRUN: NOTHING WAS CHANGED"

forgetting that you need to specify --no-dryrun for things to happen can make for some crazy debugging sessions!